### PR TITLE
fix: lite box collider shape should scale the size of box

### PR DESCRIPTION
**Pull request by yangfengzzz**
… transform

### Please check if the PR fulfills these requirements

- [ ] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

**Review by GuoLei1990 — APPROVED** [review]
+1

**Review by cptbtptpbcptdtptp — APPROVED** [review]
+1

File: packages/physics-lite/src/shape/LiteBoxColliderShape.ts [hunk 1]
```diff
@@ -41,7 +41,11 @@ export class LiteBoxColliderShape extends LiteColliderShape implements IBoxColli
    * {@inheritDoc IColliderShape.setWorldScale }
    */
   setWorldScale(scale: Vector3): void {
-    this._transform.setScale(scale.x, scale.y, scale.z);
+    const position = this._transform.position;
+    Vector3.multiply(position, scale, position);
+    this._transform.position = position;
+
+    Vector3.multiply(this._halfSize, scale, this._halfSize);
```
Comment (cptbtptpbcptdtptp) [inline packages/physics-lite/src/shape/LiteBoxColliderShape.ts:48]:
``` 
this._transform.position = this._transform.position.multiply(scale);
this._halfSize.multiply(scale);
```

Comment (yangfengzzz) [reply packages/physics-lite/src/shape/LiteBoxColliderShape.ts]: done

**Pull request merged**